cloud/kubernetes/helm/templates/pv-manifest.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if and (eq .Values.persistence.storageClass "-") (or (eq .Values.cpType "minikube") (eq .Values.cpType "aws")) }}
 {{- range $i, $vName := tuple "data-store" "logs" "rms-shared" "rms-security" "rms-webstudio" }}
-{{- if or (and (eq $vName "data-store") (eq $.Values.bsType "sharednothing")) (and (eq $vName "logs") $.Values.persistence.logs) (and (eq $vName "rms-shared") $.Values.rmsDeployment) (and (eq $vName "rms-security") $.Values.persistence.rmsSecurity $.Values.rmsDeployment) (and (eq $vName "rms-webstudio") $.Values.persistence.rmsWebstudio $.Values.rmsDeployment) }}
```

Comment:
$.Values.persistence.rmsWebstudio $.Values.rmsDeployment) is repeated twice and rms-security condition is also there

Reply:
Here rms-security is loop variable check. Condition repeats for rms-webstudio as well.